src/main/scala/com/microsoft/hyperspace/Hyperspace.scala

```diff
@@ -16,6 +16,8 @@
 
 package com.microsoft.hyperspace
 
+import scala.collection.{mutable, _}
```

Comment:
what is `_` here? It may be better to just spell it out for `scala.collection` case.

Reply:
You mean ` scala.collection.mutable.WeakHashMap[` this way?